concepts/observability-strategy/configurable-monitoring/assets/monitoring-current.drawio.svg

```diff
@@ -1,4 +1,4 @@
diagram id=&quot;X6W2UcQKy-bdUkpNeWyI&quot; name=&quot;Page-1&quot;&gt;7Vxbk5pIFP41Vu0+YEE3jfqoc0lSSXan1trazCNKq2SQNtDOaH79NpdW6O5BREBmEvMQaJrbuXx9zncO04M3692HwN6svhIHez2gO7sevO0BYAzRkP0Xjez5yMBKRpaB66Rjx4Gp+xOng3o6unUdHOYmUkI86m7yg3Pi+3hOc2N2EJCX/LQF8fJ33dhLLA1M57Ynj/7nOnSVjA7B4Dj+EbvLFb+zYY2SI2ubT07fJFzZDnnJDMG7HrwJCKHJ1np3g71IelwuyXn3rxw9PFiAfVrmhOnu4+LGmqKPD7MfoY5/kJeRpqVXeba9bfrC6cPSPZdAQLa+g6OL6D04eVm5FE839jw6+sKUzsZWdO2xPYNtLlzPuyEeCeJz4X38Y+Pyw/I744DiXWYoffgPmKwxDfZsSnpUM1AqydSWNMgHXo6aGenp2CqjlcOgnVrD8nD1o8DYRioztfxctNQd+/FpqW/Hzz9N6MF/PmugWfktUPSPjYc0IE84c8SKf9EZxKeZ8eRXk8SHgsABlAU+GCoEPoBNCRzWLHDHDlfx3CLrlaSvx7+apCyYNQQKIQOFkA2rKSGb717ImjEqKWXUGHYgSajYYWtPuksCuiJL4tve3XF0khf7cc4XQjapfL9jSvfpQmpvKcmrAu9c+i09Pdp+jLb7KN273WUO3e75js9e9xu/QLSTnDVAfP94XrzHTxTRbLEA87lK1Y41s5BVpOqQbIM5LpCmwaMDO1hiWjQxDUkiYReaToA9m7rP+UBAZQXxqeMgsPeZCRvi+jTMXPkhGjha5MF7eWSEzPzyfWL+EAkGmDzA0RwPb1LdQq23YqHdNLTBuzA0Y6A3b2mDt2JpByzMwt9jDv1axUJQ0hiR0Q1jNGDOuEAbxjWUwpnpnMUouAcsj8lsMgvY1jLa+sru5M5DpTF+sWcsuc0ZkO25S59tz5mGMFPfJApIXJY8jtMDa9dxElvFofvTnsXXi6whlRO7OJr00C0b8aLLT+z50zK2a24QPvGxHOUXhEZF7iXFS4e8OX20XjY1VcZRet8yzTQ+PM9UZFsYCiEZtPKXIItFyIxajLNqsAhDznsfguiqK7yVtX9RBnfaz8vr9uywF4wELAemFPUaqqi3sdTCkHOL8WbjMaehLvFrFf3Mng8dqBI9gKaJnGZFrxlCWqdIOAwVWYEaE/1AEv3n/dpmI9N9SPFahsNPIVMKu9fdM5PDJPKMe6YsdsIkEmhQq7ZK5IGNagvlHcVUaAu16ijyqvV5OC1Q1vjh0xQHz5Fa7j9v2WKSQud7UVDenVT6MdvUD7Be86YbbxtShXewN6V5CeclyZf6jNjTofJxhkqleaU3qSNTIFmAgspSKQk0piQ15Gnch/S/7DUOY3H9ctpCgzzkMcRR0eltakuGvH/DyJF+ZTVZuhBHKJCvVZ+ChqSENnP1Q37+mDlyMlc/L1Vn2kwfxDAKtNsm6wTTsuJp1qkbiT4UCXckVCfPm98MMcCFmgGcD4G9sH271sjpEsOonutpqoSj1VzPBN1BipO8Htt5wIHL3jxaDC7GDxVeVKhwncQFE8m4UH+F4zIzQK9FymNn7co5f7iKuDl4u1174zkl2aU55uAeSOjGZAG8nRFKyVqxdlMiuCTZUs/1mfB5m0ixQkpCfp39DRAK8ZiqEAxb9d/RNf3XOMd7D56a89P+mQXKEoxRdU9tiIGXVlIgBIwmpzb4JRKoSM+qn1tFMrea8khHxiJjUtaPbdQBFXucFsZGMWYThptdLG1+mLMcDwHZ7S9cnY/h3bBrNIewhqtIQxUENMYZInjVJbwSBPRNNMzCgKb3dTQ4DwlKhGzx+WLIUB0eSpeVkdWJAP/SsrLUrijMt2DR9Hw+IGOgWPcYCg6SqEPCQPmpgQimwoWaBlOZZaytUFUIg5csheXhzjwNdyrSXVRCfXAn84Xvszp1WEU6U51CMvn3O2worU6RJLx23GA1WGC/FLeu7FatApp1XQqmWvwGGm6s+r5db7gElrF5lQrbQNkArRsNpgwM8xFUUavV6QhqwBRojY4/s1pAZYH+SM/8RvnwSmexevYndAQ1HG1Zb6bhtTI12cVGWasZZqRrGQ1AqGj+CYeEupTS9NEg4yoCy1PWIZln9/PPZYpfPNTndaAgKtDScK/GGLtbzXfKnpWmCjLXRbI2irRXRjLQCcwSy6Y8AC1bZhXmnwoKhPBWG1SkVSAUobYxyFF0PfLub91eLgO8tCmzeamdzk3QSF9fryVcNljOBtXRxw3yhpDsVa091NqybVRJWuSg/nRNuBwwnfvVWx5GS6HOoGSkVH9zR9n1BMgJPfeZcGP7Sm5mdviaQZsneBxRNMFy9gcwWW7CLJTdVgdRmZFvI/1Pmbz5avvRV/rp7diTJndMDp7gh+p6htZe9ou7wPP93Hv9fbsARNxFa0EifQjhZfDDnyCfTWpi5bMWdJLXk80xhiUbHNhRz8L7CWMRVIcMrYSxMinsEwdreLdhq0DNXyBcScxCw1srYpX73p62M6yFlMVC2msRzxuWrdGmzXIy9w2xSEKI0/Xv+rtBsF7KF2mX8j/59yld0haKrqPG2oO46n47QlOOMHofjsArzSU9QfgU7PCnXM72BIEMMUQOtUZXuLyVskKCW8y8lTJF3kt8MjWFtdti6fVWDhL/Hm/pCmh1VP070crP/wZHK/EL54KyTTAee/Z1nIa/i5ibtx/xr9F5ub5NKZsVEKEUBlRrnM6z78aJha8Cx1WeWQeNrGon15HayGwo1MZrrJ/Jn1K0Rkw9bMMVOxhDQdhpcoq7VrfIKU1ipwQzqYWdqlT4a447F/9KUgO4ouDOW0MQoTu4akoGhZTMMMshyLmBOhKbesziaiHSzaL5536UyXaPf+42mX78q8Hw7n8=&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
Pls fix the spelling:
- `k8s` => `Kubernetes`
- `istio` => `Istio`
- Why do we say `Prometheus Operator` in the document texts, but `prometheus-operator` here?